<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionSmooth_Next"/>
    <addaction name="actionSmooth_Previous"/>
    <addaction name="actionPage_Head"/>
    <addaction name="actionPage_Tail"/>
    <addaction name="separator"/>
    <addaction name="action_Fullscreen"/>
    <addaction name="action_Hide_Menubar"/>
    <addaction name="separator"/>
    <addaction name="action_Scale_Image"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Preference"/>
   </widget>
   <widget class="QMenu" name="menu_Go">
    <property name="title">
     <string>&amp;Go</string>
    </property>
    <addaction name="action_Go_To"/>
    <addaction name="separator"/>
    <addaction name="action_Previous_Image"/>
    <addaction name="action_Next_Image"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Go"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionSmooth_Next">
   <property name="text">
    <string>Smooth &amp;Next</string>
   </property>
  </action>
  <action name="actionSmooth_Previous">
   <property name="text">
    <string>Smooth &amp;Previous</string>
   </property>
  </action>
  <action name="actionPage_Head">
   <property name="text">
    <string>Page &amp;Head</string>
   </property>
  </action>
  <action name="actionPage_Tail">
   <property name="text">
    <string>Page &amp;Tail</string>
   </property>
  </action>
  <action name="action_Fullscreen">
   <property name="text">
    <string>&amp;Full Screen</string>
   </property>
  </action>
  <action name="action_Hide_Menubar">
   <property name="text">
    <string>Hide &amp;Window</string>
   </property>
  </action>
  <action name="action_Scale_Image">
   <property name="text">
    <string>&amp;Scale Image</string>
   </property>
  </action>
  <action name="action_Preference">
   <property name="text">
    <string>&amp;Preference</string>
   </property>
  </action>
  <action name="action_Go_To">
   <property name="text">
    <string>&amp;Go To ...</string>
   </property>
  </action>
  <action name="action_Previous_Image">
   <property name="text">
    <string>&amp;Previous Image</string>
   </property>
  </action>
  <action name="action_Next_Image">
   <property name="text">
    <string>&amp;Next Image</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
